.github/workflows/get_build_targets.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+readonly TMP_OUTPUT_FILE="tmp.txt"
+
+# This limits the directory that bazel query is going to search under.
+readonly SEARCH_FOLDER="//source/common/..."
+
+set -e -o pipefail
+
+git fetch https://github.com/envoyproxy/envoy.git master 2>/dev/null
```

Comment:
How does this work? Are we inside some workflow sandbox? How does the ORIGINAL_HEAD, which I assume reflects the PR on someone's private branch, get reflected in this fetch? To improve review velocity, I'd suggest also addressing this in code comments.

Reply:
Should just be HEAD now. This was used in order to get around the git checkout HEAD^2 previously when the script was executed in later steps.

Reply:
changed ORIGINAL_HEAD to HEAD and added comment for the fetch command.

Reply:
So, to confirm, this only applies on master? I'm trying to understand how the PR under review is being included in the diff?